.github/workflows/gatk-tests.yml

```diff
@@ -0,0 +1,311 @@
+  HELLBENDER_TEST_LOGS: /hellbender-test-logs/build_reports/
+  HELLBENDER_TEST_PROJECT: broad-dsde-dev
+  TEST_VERBOSITY: minimal
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  buildDocker:
+    name: build docker images
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout gatk
+        uses: actions/checkout@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=raw,value=${{github.sha}}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+#Run our non-docker tests
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 8, 11 ]
+        experimental: [ false ]
+        scalaVersion: [ 2.11, 2.12 ]
+        testType: [ cloud, integration, unit ]
+        exclude:
+          - java: 11
+            scalaVersion: 2.11
+          - java: 8
+            scalaVersion: 2.12
+          - java: 8
+            testType: integration
+          - java: 8
+            testType: unit
+      fail-fast: false
+    continue-on-error: ${{ matrix.experimental }}
+    env:
+      TEST_TYPE: ${{ matrix.testType }}
+      SCALA_VERSION: ${{ matrix.scalaVersion }}
+    name: Java ${{ matrix.Java }} build and test ${{ matrix.testType }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: 'Set up java ${{ matrix.Java }}'
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ matrix.Java }}
+          distribution: 'adopt'
+          cache: gradle
+
+      - name: 'Grant execute permission for gradlew'
+        run: chmod +x gradlew
+
+      - name: 'Compile with Gradle'
+        run: |
+          ./gradlew compileJava ; ./gradlew installDist
+
+      #Google Cloud stuff
+      - id: 'auth'
+        uses: google-github-actions/auth@v0
+        with:
+          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+          project_id: ${{ env.HELLBENDER_TEST_PROJECT }}
+          create_credentials_file: true
+
+      - name: "export the credentials for GATK tests"
+        run: echo "HELLBENDER_JSON_SERVICE_ACCOUNT_KEY=${{ steps.auth.outputs.credentials_file_path }}" >> $GITHUB_ENV
+
+      - name: 'Set up Cloud SDK'
+        uses: google-github-actions/setup-gcloud@v0
+
+      - name: pull lfs files
```

Comment:
should we put this after the compile so in compile error times we fail faster?

Reply:
I thought the tests require lfs files to compile?

Reply:
Compilation only needs main large, not test large.  Not a big deal either way though, lets punt.